disnake/ext/commands/bot.py

```diff
@@ -264,12 +358,84 @@ class InteractionBot(InteractionBotBase, disnake.Client):
         .. versionadded:: 2.5
     """
 
-    pass
+    if TYPE_CHECKING:
+
+        def __init__(
+            self,
+            *,
+            owner_id: Optional[int] = None,
+            owner_ids: Set[int] = ...,
+            reload: bool = False,
+            sync_commands: bool = True,
+            sync_commands_debug: bool = False,
+            sync_commands_on_cog_unload: bool = True,
+            sync_permissions: bool = False,
```

Comment:
removed in 2.5

Reply:
Fixed in 1a642234474e2777ce36aa5d754ca488f2cd2024.